File: Assets/Scripts/Checkpoint.js
#pragma strict

var SpawnPoint : Transform;

function OnTriggerEnter(col : Collider)
{
	if(col.tag =="Boat")
	{
		SpawnPoint.position = Vector3(transform.position.x, transform.position.y, transform.position.z);
	
	}

}

File: Assets/Scripts/checkpoint.cs
using UnityEngine;
using System.Collections;
//chiedere come far diventare la posizione del respawn uguale alla posizione del checkpoint.
public class checkpoint : MonoBehaviour 
{
	public Transform Check;
	public Transform RespawnPoint;

	void OnTriggerEnter (Collider coll)
	{
		if (coll.tag == "Player")
		{
			RespawnPoint.transform.position = Check.position;
		}
	}
}
